### ISSUE TITLE
함수 수정하기

### ISSUE THREAD

**Issue by LeeMineo** [2022miniproject/studyCafe#8]
안녕하세요! 저는 제가 구현한 기능중에서 시간추가 함수를 구현하려고 합니다!

**Comment by LeeMineo** [comment]
아 그리고 경민님이 개발하신 부분 실행을 해보았는데 5번을 누르면 메뉴창이 안뜨고 메뉴를 선택하세요가 뜹니다! 그 부분 수정 부탁 드립니다.

**Comment by ChrongZilla** [comment]
네! 5번 눌렀을 때 메뉴가 나오도록, 그리고 5-2에 주문할 때 음식 리스트 나오도록 수정 완료했고 그 외에 다른 부분이랑 다른 오류 뜨는 코드들 수정해서 merge 하겠습니다 !
